scripts/docker/clean-logs.sh

```diff
@@ -32,7 +33,7 @@ while true; do
   echo "Trimming airflow logs to ${RETENTION} days."
   find "${DIRECTORY}"/logs \
     -type d -name 'lost+found' -prune -o \
-    -type f -mtime +"${RETENTION}" -name '*.log' -print0 | \
+    -type f -mtime -"${ABS_RETENTION}" -name '*.log' -print0 | \
```

Comment:
The existing behavior is changed. we usually look for the files for which are not modified since x days. This is not matching with the given expression.

Reply:
Hi @dirrao , original script requires negative retention (x day). What I change is to make the given x to absolute value to match the description from you said.

Reply:
> The existing behavior is changed. we usually look for the files for which are not modified since x days. This is not matching with the given expression.

Original will ask to give `-x` day, or it will not work. After my change, no matter given x day or -x day, the script will find the files which are not modified since |x| days ago.